package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node",
```

Comment:
Don't be afraid. `ts-node` has some magic under the hood that makes it performatic for production.
`"start": "ts-node src/app.ts"`